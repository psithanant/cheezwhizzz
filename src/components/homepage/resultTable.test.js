import React from 'react';
import ReactDOM from 'react-dom';
import { shallow } from 'enzyme';
import ResultTable from './resultTable';
import toJson from 'enzyme-to-json';

const TEST_CHEESE_LIST = [
  { name: "brie", firmness: "soft", animal: "cow", favorite: 0, id: 1 }
];

describe('ResultTable', () => {
  it('renders', () => {
    const div = document.createElement('div');
    ReactDOM.render(<ResultTable cheeses={TEST_CHEESE_LIST} />, div);
  });

  test('snapshots', () => {
    const component = shallow(
      <ResultTable cheeses={TEST_CHEESE_LIST} />,
    );
    expect(toJson(component)).toMatchSnapshot();
  });
});
