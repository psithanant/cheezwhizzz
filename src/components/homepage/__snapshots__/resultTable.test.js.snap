// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ResultTable snapshots 1`] = `
<table
  className="table-striped table-bordered table"
>
  <thead>
    <tr>
      <th>
        Name
      </th>
      <th>
        Firmness
      </th>
      <th>
        Animal
      </th>
      <th>
        Favorite
      </th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>
        brie
      </td>
      <td>
        soft
      </td>
      <td>
        cow
      </td>
      <td>
        <input
          type="checkbox"
        />
      </td>
    </tr>
  </tbody>
</table>
`;
